app/clients/letter/dvla.py

```diff
@@ -269,3 +270,23 @@ def _build_unstructured_address(address_without_recipient):
         unstructured_address["postcode"] = postcode
 
         return unstructured_address
+
+    @staticmethod
+    def _build_international_address(address_without_recipient):
+        address_line_keys = ["line1", "line2", "line3", "line4", "line5"]
+
+        country = address_without_recipient[-1]
+        address_without_country = address_without_recipient[:-1]
+
+        unstructured_address = dict(zip(address_line_keys, address_without_country))
```

Comment:
Optional - this variable could be named `international_address` since it provides the value for the `internationalAddress` key

Reply:
/doh 👍